.github/workflows/prettier.yml

```diff
@@ -10,13 +10,17 @@ jobs:
       - uses: actions/setup-node@v2
         with:
           node-version: '14'
+      - run: npm ci
+      - name: Fail CI
+        if: ${{ github.event_name == 'pull_request' || github.event_name == 'pull_request_target' }}
```

Comment:
I think this should run for all changes

Reply:
Hmm it will commit if it `push` so no need to fail